client/src/components/Stripe-button/stripe-button.js

```diff
@@ -0,0 +1,44 @@
+import React from "react"
+import StripeCheckout from "react-stripe-checkout"
+import axios from "axios"
+//stripe sees data in cents not dollars
+//submission getting handled by this compothe on success callback that triggers is when we submit
+const StripeCheckoutButton = ({ price }) => {
+  const priceForStripe = price * 100
```

Comment:
Don't ever do currency math in JavaScript. Same reason Stripe is in cents in the first place! You should use a library that is built for that.